h2/src/main/org/h2/api/IntervalQualifier.java

```diff
@@ -0,0 +1,337 @@
+/*
+ * Copyright 2004-2018 H2 Group. Multiple-Licensed under the MPL 2.0,
+ * and the EPL 1.0 (http://h2database.com/html/license.html).
+ * Initial Developer: H2 Group
+ */
+package org.h2.api;
+
+/**
+ * Interval qualifier.
+ */
+public enum IntervalQualifier {
```

Comment:
This is actually an "interval subtype" as you originally named it. Are we following the SQL standard naming here? Just curious.

Reply:
This is a name from the standard.